docs/subscribe-flow.md

```diff
@@ -53,55 +53,80 @@ Another way to trigger the subscribe flow is by starting [Offers flow](./offers-
+| data          | Object    | The information contained within the ID Token.                                                                                                                                                                                                                                                          |
+| id            | String    | The user’s Google Sign-in ID. This corresponds to the sub field of the idToken.                                                                                                                                                                                                                         |
+| email         | String    | The user’s email address. <blockquote><b>Note:</b> A Google account's email address can change, so don't use it to identify a user. Instead, use the account's ID, which you can get on the client with getBasicProfile().getId() , and on the backend from the sub claim of the ID token.</blockquote> |
+| emailVerified | Boolean   | Returns true if the email address is verified by Google.                                                                                                                                                                                                                                                |
+| name          | String    | Full name, including given and family name.                                                                                                                                                                                                                                                             |
+| givenName     | String    | The user's first name.                                                                                                                                                                                                                                                                                  |
+| familyName    | String    | The user's last name.                                                                                                                                                                                                                                                                                   |
+| pictureUrl    | String    | The user's profile picture.                                                                                                                                                                                                                                                                             |
+
+### `entitlements`
+
+The entitlement object will contain the entitlement for the successful purchase, it won't have all the user entitlements. It has the same structure of the [Entitlements Flow response](./entitlements-flow.md#entitlement-response).
```

Comment:
nit: prefer period: "...for the successful purchase. It won't have all the user..."

Reply:
:+1: